pkg/interactive/interactive_client.go

```diff
@@ -345,8 +346,7 @@ func (c *InteractiveClient) runInteractivePrompt(ctx context.Context) (ret utils
 			ASCIICode: constants.AltRightArrowASCIICode,
 			Fn:        prompt.GoRightWord,
 		}),
-		// ignore suppressed ASCII codes
-		prompt.OptionAddASCIICodeBind(c.suppressOnInput()...),
+		prompt.OptionBufferPreHook(cleanBufferForWSL),
```

Comment:
although it's cleaner, I dopn;t really like calling `cleanBufferForWSL` if it is not WSL - I'd rather do the check first